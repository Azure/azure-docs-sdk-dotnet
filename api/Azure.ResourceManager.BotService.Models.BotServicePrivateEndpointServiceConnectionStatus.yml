### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus&text=BotServicePrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.op_Equality(Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus,Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.op_Inequality(Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus,Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Equals(Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus
commentId: T:Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus
namespace: Azure.ResourceManager.BotService.Models
name: BotServicePrivateEndpointServiceConnectionStatus
fullName: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus
nameWithType: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.BotService.dll
packagesWithMoniker:
- value: Azure.ResourceManager.BotService v1.0.1
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.BotService v1.1.0-beta.1
  monikers:
  - azure-dotnet-preview
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct BotServicePrivateEndpointServiceConnectionStatus : IEquatable<Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type BotServicePrivateEndpointServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure BotServicePrivateEndpointServiceConnectionStatus

      Implements IEquatable(Of BotServicePrivateEndpointServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
summary: The private endpoint connection status.
metadata:
  api_name:
  - Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.BotService.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus
  - Azure::ResourceManager::BotService::Models::BotServicePrivateEndpointServiceConnectionStatus
  - BotServicePrivateEndpointServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
