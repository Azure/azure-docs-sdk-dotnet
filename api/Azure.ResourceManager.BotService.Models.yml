### YamlMime:NetNamespace
classes:
- uid: Azure.ResourceManager.BotService.Models.AcsChatChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.AlexaChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.AlexaChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.ArmBotServiceModelFactory
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotChannelGetWithKeysResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotChannelRegenerateKeysContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotChannelSettings
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotChannelSite
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotConnectionSettingParameter
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotConnectionSettingProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotCreateEmailSignInUriResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServiceHostSettingsResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServiceNameAvailabilityContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServiceNameAvailabilityResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateLinkResourceData
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateLinkServiceConnectionState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServiceProvider
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServiceProviderParameter
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServiceProviderProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServiceSku
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.DirectLineChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.DirectLineChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.DirectLineSite
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.DirectLineSpeechChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.DirectLineSpeechChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.EmailChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.EmailChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.FacebookChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.FacebookChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.FacebookPage
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.GetBotServiceQnAMakerEndpointKeyContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.GetBotServiceQnAMakerEndpointKeyResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.KikChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.KikChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.LineChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.LineChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.LineRegistration
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.M365Extensions
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.MsTeamsChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.MsTeamsChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.Omnichannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.OutlookChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.SearchAssistant
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.SkypeChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.SkypeChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.SlackChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.SlackChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.SmsChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.SmsChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.TelegramChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.TelegramChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.TelephonyChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.TelephonyChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.TelephonyChannelResourceApiConfiguration
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.TelephonyPhoneNumbers
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.WebChatChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.WebChatChannelProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.WebChatSite
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
enums:
- uid: Azure.ResourceManager.BotService.Models.BotServiceKey
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.EmailChannelAuthMethod
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.RegenerateKeysBotChannelName
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
structs:
- uid: Azure.ResourceManager.BotService.Models.BotChannelName
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotMsaAppType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServiceKind
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointConnectionProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServicePublicNetworkAccess
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServiceSkuName
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.BotService.Models.BotServiceSkuTier
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.BotService.Models
commentId: N:Azure.ResourceManager.BotService.Models
name: Azure.ResourceManager.BotService.Models
fullName: Azure.ResourceManager.BotService.Models
devLangs:
- csharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  f1_keywords:
  - Azure.ResourceManager.BotService.Models
  - Azure::ResourceManager::BotService::Models
  - Models
  - BotService.Models
  - BotService::Models
  helpviewer_keywords:
  - Azure.ResourceManager.BotService.Models namespace [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
