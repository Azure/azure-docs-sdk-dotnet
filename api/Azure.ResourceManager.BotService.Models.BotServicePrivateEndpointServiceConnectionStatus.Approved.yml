### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus&text=BotServicePrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Approved
  nameWithType: BotServicePrivateEndpointServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As BotServicePrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.BotService.Models
name: Approved
fullName: Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Approved
nameWithType: BotServicePrivateEndpointServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.BotService.dll
packagesWithMoniker:
- value: Azure.ResourceManager.BotService v1.0.1
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.BotService v1.1.0-beta.1
  monikers:
  - azure-dotnet-preview
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Approved
  - Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.BotService.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.BotService.Models.BotServicePrivateEndpointServiceConnectionStatus.Approved
  - Azure::ResourceManager::BotService::Models::BotServicePrivateEndpointServiceConnectionStatus::Approved
  - Approved
  - BotServicePrivateEndpointServiceConnectionStatus.Approved
  - BotServicePrivateEndpointServiceConnectionStatus::Approved
  helpviewer_keywords:
  - BotServicePrivateEndpointServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct BotServicePrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
